custom_components/frigate/media_source.py

```diff
@@ -47,7 +47,12 @@
 
 async def async_get_media_source(hass: HomeAssistant) -> MediaSource:
     """Set up Frigate media source."""
-    return FrigateMediaSource(hass)
+    config_entry = hass.config_entries.async_entries(DOMAIN)[0]
```

Comment:
This stumbles into a limitation: we can only turn the media browser on or off for ALL Frigate instances in a given HA installation. As is, this will only respect the option for the _first_ configured Frigate installation. 

I would suggest we only show the media browser when ALL config entries have CONF_MEDIA_BROWSER_ENABLE set true. When a single instance has the option set the false, we default to not showing the media browser.

Reply:
Ah I see, that makes sense then, will work on this 👍